### PR TITLE
Use Modified NgraphBackend class within test utilities.

### PR DESCRIPTION
**Pull request by arogowie-intel**
Code in this PR is on branch `checkout`ed from #82 and this code should be merged after #82 is merged.

- Use of modified `NgraphBackend` class from #82 makes utility function simpler and cleans-up duplicate code.

### CODE REVIEW

File: ngraph_onnx/onnx_importer/backend.py [hunk 1]
```diff
@@ -79,7 +127,7 @@ def __init__(self, ng_model, device='CPU'):  # type: (List[Dict], str) -> None
         super(NgraphBackendRep, self).__init__()
         self.device = device
```
Comment (tsocha) [inline ngraph_onnx/onnx_importer/backend.py]: Maybe use backend instead of device?

File: tests/test_ops_nonlinear.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def leaky_relu(x, alpha=0.01):
     assert_onnx_import_equals_callable('LeakyRelu', leaky_relu, [-2, -1., 0., 1., 2.], alpha=0.5)
     assert_onnx_import_equals_callable('LeakyRelu', leaky_relu, [0.])
     assert_onnx_import_equals_callable('LeakyRelu', leaky_relu,
-                                       [-0.9, -0.8, -0.7, -0.4, -0.3, -0.2, -0.1], alpha=1)
+                                       [-0.9, -0.8, -0.7, -0.4, -0.3, -0.2, -0.1], alpha=1.)
```
Comment (tsocha) [inline tests/test_ops_nonlinear.py]: Why change from int to float?

Comment (arogowie-intel) [reply tests/test_ops_nonlinear.py]: Onnx model.check fails. According to Op documentation this value should be floating point.

File: ngraph_onnx/onnx_importer/backend.py (hunk 1 above)
Comment (arogowie-intel) [reply ngraph_onnx/onnx_importer/backend.py]: Updated in #82

**Pull request merged**